Buy/Utilities/Header.swift

```diff
@@ -34,4 +34,5 @@ internal struct Header {
     static var sdkVersion    = "X-SDK-Version"
     static var sdkVariant    = "X-SDK-Variant"
     static var queryTag      = "X-Query-Tag"
+    static var sdkLocale     = "Accept-Language"
```

Comment:
Can we name this after the header? `language` identifies the header more accurately.